__tests__/app.test.js

```diff
@@ -91,3 +91,53 @@ describe('GET /api/articles', () => {
     });
 })
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('responds with status 200 and sends comments with correct keys and values', () => {
+        return request(app).get('/api/articles/9/comments').expect(200).then((res) => {
+            const comments = res.body.comments;
+            const expectComment = {
+                body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+                votes: 16,
+                author: "butter_bridge",
+                article_id: 9,
+                created_at: "2020-04-06T12:17:00.000Z",
+              }
+            
+            expect(comments).toEqual(expect.arrayContaining([expect.objectContaining(expectComment)]))
+        }) 
+    });
+
+    test('does not send comments from other articles, sends all comments for requested article', () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+            const comments = res.body.comments;
+            expect(comments).toHaveLength(11);
+            comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1)
+            })
+        })
+    });
+
+    test('comments are sorted by date with most recent first', () => {
+        return request(app).get('/api/articles/1/comments')
+        .then((res) => {
+            const comments = res.body.comments;
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+        })
+    });
+
+    test('responds with status 404 and appropriate message when given a valid, but non-existent article id', () => {
+        return request(app).get('/api/articles/83/comments').expect(404).then((res) => {
+            expect(res.body.msg).toBe('Article does not exist');
+        })
+    });
+
+    test('responds with status 204 when given a valid article_id that has no associated comments ', () => {
+        return request(app).get('/api/articles/2/comments').expect(204)
```

Comment:
I feel that a 204 is not the correct status. An intentional empty is still fine. This should be a 200 status code